src/storage/mvcc/reader/point_getter.rs

```diff
@@ -554,39 +557,47 @@ mod tests {
         must_get_none(&mut getter, b"foo1");
         let s = getter.take_statistics();
         assert_seek_next_prev(&s.write, 1, 0, 0);
+        assert_eq!(s.processed_size, 0);
         // Get again
         must_get_none(&mut getter, b"foo1");
         let s = getter.take_statistics();
         assert_seek_next_prev(&s.write, 1, 0, 0);
+        assert_eq!(s.processed_size, 0);
 
         // Get a key that exists
         must_get_value(&mut getter, b"foo2", b"foo2v");
         let s = getter.take_statistics();
         // We have to check every version
         assert_seek_next_prev(&s.write, 1, 40, 0);
+        assert_eq!(s.processed_size, 269);
```

Comment:
Could we describe the key structure here? `269` seems too magic and difficult to maintain if the next contributor doesn't familiar with key content.